bundles/python/python_mode.moon

```diff
@@ -0,0 +1,34 @@
+-- Copyright 2012-2015 The Howl Developers
+-- License: MIT (see LICENSE.md at the top-level directory of the distribution)
+
+{
+  lexer: bundle_load('python_lexer')
+
+  comment_syntax: '#'
+
+  default_config:
+    word_pattern: r'\\b[\\w_][\\w\\d_]+\\b'
```

Comment:
Yes, that would also show up in the lexer for the alpha matcher. It's hard to say what the best way of handling this would be, perhaps a final case for non-matched tokens that would classify them as identifiers (e.g. if not a number, string, etc..). Correctly checking unicode character classes seems like it would be to much work and have performance implications, but perhaps that's doable as well.